tests/jballerina-unit-test/src/test/resources/test-src/expressions/binaryoperations/type-test-expr.bal

```diff
@@ -224,6 +224,28 @@ function testSealedRecordTypes() returns [boolean, boolean] {
     return [a is A3, a is B3];
 }
 
+type Country record {|
+    string code?;
+    string name?;
+    record {|
+        string code?;
+        string name?;
+    |} continent?;
+|};
+
+type MyCountry record {|
```

Comment:
Shall we check the behavior for defautable field values?

Reply:
I will look into this.

Reply:
I cannot assign default values for optional fields because it is not allowed by the spec. So I added one for a required field in the nested record.